.eslintrc.js

```diff
@@ -35,7 +35,15 @@ module.exports = {
     "func-names" : ["warn", "as-needed"],
     "no-underscore-dangle": ["error", {
       "allowAfterThis": true
-    }]
+    }],
+    "quotes" : ["error", "single", { "allowTemplateLiterals": true }],
+    "arrow-parens" : "off",
+    "arrow-body-style" : "off",
+    "func-names" : "off",
+    "prefer-arrow-callback" : "off",
+    "no-underscore-dangle" : "off",
+    "no-unused-vars" : "off",
```

Comment:
`next` isn't required.  The following code is perfectly valid expressjs middleware:
```js
app.get('/route', (req, res) => res.redirect('/other/route'));
```

If code doesn't use `next`, it is confusing to pass it into a function.  A better fix would be to simply remove `next` from the parameters.

As mentioned about, if it is error handling middleware (which we should only have one), we can just put comment telling `ESlint` to ignore that line.